src/Visualization/Python/PlotTrajectories.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Plot trajectory from inspiral run, concatenating all segments
+
+import click
+import h5py
+import matplotlib.pyplot as plt
+import numpy as np
+
+from spectre.Visualization.Plot import (
+    apply_stylesheet_command,
+    show_or_save_plot_command,
+)
+
+
+@click.command(name="trajectories")
+@click.argument(
+    "h5_files",
+    nargs=-1,
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+)
+@click.option(
+    "--subfile-name-aha",
+    "-A",
+    default="ApparentHorizons/ControlSystemAhA_Centers.dat",
+    help=(
+        "Name of subfile containing the apparent horizon centers for object A."
+    ),
+)
+@click.option(
+    "--subfile-name-ahb",
+    "-B",
+    default="ApparentHorizons/ControlSystemAhB_Centers.dat",
+    help=(
+        "Name of subfile containing the apparent horizon centers for object B."
+    ),
+)
+@show_or_save_plot_command()
+@apply_stylesheet_command()
+def plot_trajectories_command(h5_files, subfile_name_aha, subfile_name_ahb):
+    """Plot trajectories in inspiral simulation
+
+    Concatenates partial trajectories from each h5 reductions files and plots
+    full trajectories.
+    """
+    reductionfiles = h5_files
+    AhA, AhB = import_A_and_B(
+        reductionfiles, subfile_name_aha, subfile_name_ahb
+    )
+    if AhA is not None and AhB is not None:
+        plot_trajectory(AhA, AhB)
+    else:
+        click.echo("Error: Unable to import data or not valid data.")
+
+
+def import_A_and_B(filenames, subfile_name_aha, subfile_name_ahb):
+    A_data = []
+    B_data = []
+
+    for filename in filenames:
+        with h5py.File(filename, "r") as file:
+            if subfile_name_aha in file:
+                A_data.append(np.array(file[subfile_name_aha][:]))
+            if subfile_name_ahb in file:
+                B_data.append(np.array(file[subfile_name_ahb][:]))
+
+    if A_data and B_data:
+        A = np.concatenate(A_data)
+        B = np.concatenate(B_data)
+        min_length = min(len(A), len(B))
+        return A[:min_length], B[:min_length]
+    else:
+        return None, None
+
+
+def plot_trajectory(AhA, AhB, sample_rate=15):
```

Comment:
Please add documentation to this function that describes what the function does and also describes each argument. You find examples in the other Python files in this directory.